File: Python/Count Primes/Count Primes.py
class Solution:
    def countPrimes(self, n: int) -> int:
        chk_prime = [True for i in range(n-1)]
        chk_prime.insert(0, False)
        for i in range(2, n):
            if(not chk_prime[i]):
                continue
            j = i*i
            while(j < n):
                chk_prime[j] = False
                j += i
        p_count = 0
        for i in range(n):
            if(chk_prime[i]):
                p_count += 1
        return p_count

File: C#/Count Primes/Count Primes.cs
public class Solution {
    public int CountPrimes (int n) {
        int primeCount = 0;
        for (int i = 1; i < n; i++)
            if (IsPrime (i)) primeCount++;
        return primeCount;
    }

    public bool IsPrime (int n) {
        if (n <= 1) return false;
        for (int i = 2; i * i <= n; i++)
            if (n % i == 0) return false;
        return true;
    }
}